changelog.md

```diff
@@ -1,3 +1,111 @@
+
+## CLI
+
+* list default categories in help text ([#9399](https://github.com/googlechrome/lighthouse/pull/9399))
+
+## Clients
+
+* devtools: fix collapsing-width svg in flexbox ([#9602](https://github.com/googlechrome/lighthouse/pull/9602))
+* devtools: update test command to run only .js files ([#9532](https://github.com/googlechrome/lighthouse/pull/9532))
+* proto: don't filter channel from configSettings ([#9554](https://github.com/googlechrome/lighthouse/pull/9554))
+* viewer: remove polyfills ([#9553](https://github.com/googlechrome/lighthouse/pull/9553))
+
+## Deps
+
+* chrome-launcher: update to 0.11.2 ([#9515](https://github.com/googlechrome/lighthouse/pull/9515))
+* details-element-polyfill@2.4.0 ([#9552](https://github.com/googlechrome/lighthouse/pull/9552))
+* update jsonlint-mod to 1.7.5 ([#9465](https://github.com/googlechrome/lighthouse/pull/9465))
+
+## Docs
+
+* fix broken plugin example link ([#9524](https://github.com/googlechrome/lighthouse/pull/9524))
+* lightwallet: add link to budget.json repo ([#9579](https://github.com/googlechrome/lighthouse/pull/9579))
+
+## I18n
+
+* import translations from tc/ ([#9577](https://github.com/googlechrome/lighthouse/pull/9577))
+* add registerLocaleData() method ([#9638](https://github.com/googlechrome/lighthouse/pull/9638))
+* rewrite aria-required-children titles ([#9590](https://github.com/googlechrome/lighthouse/pull/9590))
+* import translated strings ([#9620](https://github.com/googlechrome/lighthouse/pull/9620))
+* reword pwa-page-transitions description ([#9615](https://github.com/googlechrome/lighthouse/pull/9615))
+* throw on excess placeholder replacement values ([#9580](https://github.com/googlechrome/lighthouse/pull/9580))
+* remove translated messages when ICU arguments change ([#9598](https://github.com/googlechrome/lighthouse/pull/9598))
+* use better types for intl-messageformat ([#9570](https://github.com/googlechrome/lighthouse/pull/9570))
+* add code spans to uses-rel-preconnect description ([#9568](https://github.com/googlechrome/lighthouse/pull/9568))
+* fix bugs in locale-importing scripts ([#9621](https://github.com/googlechrome/lighthouse/pull/9621))
+* fix custom formatted ICU within plurals ([#9460](https://github.com/googlechrome/lighthouse/pull/9460))
+* capitalize robots.txt displayValue ([#9567](https://github.com/googlechrome/lighthouse/pull/9567))
+* add link to cldr parentLocales ([#9520](https://github.com/googlechrome/lighthouse/pull/9520))
+* move pwa audits to i18n system ([#9105](https://github.com/googlechrome/lighthouse/pull/9105))
+* use tsc to collect UIStrings ([#9487](https://github.com/googlechrome/lighthouse/pull/9487))
+* align type names with docs ([#9461](https://github.com/googlechrome/lighthouse/pull/9461))
+* expand placeholder support ([#9114](https://github.com/googlechrome/lighthouse/pull/9114))
+* roll: roll es-419, fr, id, zh-TW locales. correct fr ([#9454](https://github.com/googlechrome/lighthouse/pull/9454))
```

Comment:
maybe combine the `import translations`/`import translated`/`roll` prs into one entry?